src/test/java/nz/cri/gns/NZSHM22/opensha/inversion/MFDManipulationTest.java

```diff
@@ -86,9 +86,9 @@ public void TestAddMfdUncertainty() {
             dist.set(i, i);
         }
 
-        UncertainIncrMagFreqDist actual = MFDManipulation.addMfdUncertainty(dist, 5.1, 20.0, 0.5);
+        UncertainIncrMagFreqDist actual = MFDManipulation.addMfdUncertainty(dist, 5.1, 20.0, 0.5, 0.9);
 
-        assertEquals(List.of(1.0E-20, 0.22360679774997896, 0.3162277660168379, 0.3872983346207417, 0.4472135954999579, 0.5, 0.5477225575051661, 0.5916079783099616, 0.6324555320336758, 0.6708203932499369, 0.7071067811865475, 0.7416198487095662, 0.7745966692414834, 0.8062257748298549, 0.8366600265340755, 0.8660254037844385, 0.8944271909999159, 0.9219544457292888, 0.9486832980505138, 0.9746794344808963, 1.0, 1.0246950765959597, 1.0488088481701514, 1.0723805294763609, 1.0954451150103321, 1.1180339887498947, 1.1401754250991378, 1.1618950038622249, 1.1832159566199232, 1.2041594578792296, 1.2247448713915892, 1.2449899597988732, 1.2649110640673515, 1.2845232578665127, 1.3038404810405297, 1.3228756555322954, 1.3416407864998738, 1.3601470508735443, 1.378404875209022, 1.396424004376894),
+        assertEquals(List.of(1.0E-20, 4.024922359499621, 5.692099788303082, 6.971370023173351, 8.049844718999243, 9.0, 9.859006035092989, 10.648943609579307, 11.384199576606164, 12.074767078498864, 12.727922061357855, 13.349157276772193, 13.942740046346701, 14.512063946937388, 15.059880477613358, 15.588457268119893, 16.099689437998485, 16.595180023127195, 17.076299364909246, 17.544229820656135, 18.0, 18.444511378727274, 18.878559267062727, 19.302849530574495, 19.718012070185978, 20.124611797498105, 20.52315765178448, 20.91411006952005, 21.297887219158614, 21.67487024182613, 22.045407685048602, 22.409819276379718, 22.76839915321233, 23.12141864159723, 23.469128658729534, 23.811761799581316, 24.149534156997728, 24.482646915723798, 24.811287753762397, 25.135632078784095),
```

Comment:
@voj I assume thsee changes are the same modular loading issue we discussed on the maxMag PR